### PR TITLE
Corrected mariadb services for docker-compose.yml 

### PR DESCRIPTION
**Pull request by pugnator-12**
mariadb services for docker-compose.yml in "without searchguard" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1464,20 +1464,25 @@ version: '2.2'
...
+      command: --wait_timeout=2592000 --interactive_timeout=2592000 --max_connections=300
 
     elasticsearch:
       image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.1.4
       command: /elasticsearch/bin/elasticsearch -E network.bind_host=0.0.0.0
       ports:
         - 9200:9200
       environment:
-        - ES_JAVA_OPTS=-Xms2g -Xmx2g
+        - ES_JAVA_OPTS=-Xms1g -Xmx1g
```
Comment (valeriocos) [inline README.md]: please remove this change

File: README.md [hunk 2]
```diff
@@ -1464,20 +1464,25 @@ version: '2.2'
 
 services:
```
Comment (valeriocos) [inline README.md]: can you align the structure of this docker-compose to the one with searchguard? elasticsearch should be the first entry, followed by kibiter and mariadb. Thanks

**Comment by pugnator-12** [comment]
@valeriocos Thank you! I have made the necessary changes.

**Comment by valeriocos** [comment]
Thank you @pugnator-12 for the quick reaction, can you squash the commits in one?

**Comment by pugnator-12** [comment]
Thanks!

**Pull request merged**